pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml

```diff
@@ -479,7 +396,7 @@ state-facts: true
       %div.policy-item
         - if state_info[:state_plan_s] == 'Yes'
           %div
-            <i class="far fa-check-square"></i>
```

Comment:
Did these not work before?

Reply:
No, the checked squares are showing the invalid icon symbol right now
<img width="115" alt="Screen Shot 2021-01-13 at 3 56 17 PM" src="https://user-images.githubusercontent.com/17147070/104520160-8097c580-55af-11eb-8b3d-c9200b4a4a1f.png">